cli/awsbatch/awsbhosts.py

```diff
@@ -156,6 +168,20 @@ def __create_host_item(container_instance, ec2_instance):
                 if attr["name"] == "ecs.instance-type":
                     instance_type = attr["value"]
                     break
+            cpu_registered = "-"
+            mem_registered = "-"
+            for resource in container_instance["registeredResources"]:
```

Comment:
You can maybe group them in a single for loop (something like for resource in ["registeredResoures", "remainingResources"]) or extract this into a function